aimmo-game/simulation/avatar/avatar_manager.py

```diff
@@ -23,8 +23,39 @@ def remove_avatar(self, user_id):
 
     @property
     def avatars(self):
-        return self.avatars_by_id.viewvalues()
+        return list(self.avatars_by_id.viewvalues())
```

Comment:
Why do we cast to list?

Reply:
Okay good point. Maybe I didn't think it through well enough as to how this may affect everything else, but it doesn't seem to cause an issue and it was the easiest way to iterate through the avatar JSON side and then append it to our list which is inserted in the `players` top level of serialisation.

Here's what I needed it for:
![code](https://i.imgur.com/INGTxOe.png)

Reply:
I'd consider using itervalues instead of viewvalues here.

Reply:
How come @CelineBoudier?

Reply:
https://stackoverflow.com/questions/21783763/in-python-2-what-are-the-performance-trade-offs-for-viewvalues-viewitems-ov @OlafSzmidt